sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3318,6 +3318,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val BASE_64_CHUNKING = buildConf("spark.sql.base64.chunking")
+    .doc("When true, base64 strings generated by the base64 function are chunked into lines of " +
+      "at most 76 characters. When false, the base64 strings are not chunked.")
+    .version("3.5.2")
+    .booleanConf
+    .createWithDefault(true)
```

Comment:
Since we decide to keep the existing behavior as the default, could you revise the PR title and description, @ted-jenks ?